geode-gfsh/src/integrationTest/java/org/apache/geode/management/internal/cli/GfshParserAutoCompletionIntegrationTest.java

```diff
@@ -410,7 +410,7 @@ public void testObtainHintForData() {
     String hintArgument = "data";
     String hintsProvided = gfshParserRule.getCommandManager().obtainHint(hintArgument);
     String[] hintsProvidedArray = hintsProvided.split(lineSeparator());
-    assertThat(hintsProvidedArray.length).isEqualTo(17);
+    assertThat(hintsProvidedArray.length).isEqualTo(18);
```

Comment:
Better syntax is:
```
assertThat(hintsProvidedArray).hasSize(18);
```
This way if it fails, `hintsProvidedArray` will be fully printed out instead of just the integer of its length.